### DISCUSSION TITLE
Orthographic camera - 2D side-scrolling games

### DISCUSSION THREAD

**Discussion by AttackButton** [lllyasviel/ControlNet#403] [Ideas]
Since it is difficult to keep the images of a room/dungeon or landscape in stable diffusion always in the desired perspective to create a 2D game, such as metroidvanias or platformers, I think it would be a good addition to force the model to keep a 2D perspective with controlnet. Like the "orthographic camera", name used by Unity engine for this type of perspective.

**Comment by geroldmeisinger** [comment]
similar idea here https://github.com/lllyasviel/ControlNet/issues/238

**Comment by geroldmeisinger** [comment]
If you only need orthographic projection I think you might be better off training Loras, see here https://civitai.com/models/23240/dandd-battlemaps
